### PR TITLE
pub:fluid

### PR DESCRIPTION
**Pull request by AmberLJC**

### CODE REVIEW

File: source/publications/index.md [hunk 1]
```diff
@@ -21,6 +21,17 @@ venues:
   # key is also used in display
   Conferences:
     # conference short name
+    MLSys'22:
```
Comment (Aetf) [inline source/publications/index.md:24]:
```suggestion
    MLSys'21:
```

File: source/_data/SymbioticLab.bib [hunk 2]
```diff
@@ -626,3 +629,25 @@ @InProceedings{kayak:nsdi21
...
+  author    = {Peifeng Yu*, Jiachen Liu* and Mosharaf Chowdhury},
+  booktitle = {MLSys},
+  title     = {Fluid: Resource-Aware Hyperparameter Tuning Engine},
+  year      = {2021},
+  publist_confkey = {MLSys'21},
+  publist_link = {paper || fluid-mlsys21.pdf},
+  publist_link = {slides || fluid-mlsys21-talk.pdf},
+  publist_badge = {Artifacts Available},
+  publist_badge = {Artifacts Evaluated Functional},
+  publist_badge = {Artifacts Replicated},
```
Comment (Aetf) [inline source/_data/SymbioticLab.bib:643]:
```suggestion
```

There's no artifacts evaluation yet. We can add these later.

File: source/_data/SymbioticLab.bib [hunk 3]
```diff
@@ -626,3 +629,25 @@ @InProceedings{kayak:nsdi21
...
 }
+
+@InProceedings{fluid:mlsys21,
+  author    = {Peifeng Yu*, Jiachen Liu* and Mosharaf Chowdhury},
+  booktitle = {MLSys},
+  title     = {Fluid: Resource-Aware Hyperparameter Tuning Engine},
+  year      = {2021},
+  publist_confkey = {MLSys'21},
+  publist_link = {paper || fluid-mlsys21.pdf},
+  publist_link = {slides || fluid-mlsys21-talk.pdf},
```
Comment (Aetf) [inline source/_data/SymbioticLab.bib:640]: Upload the `pptx` file

File: source/publications/index.md [hunk 4]
```diff
@@ -21,6 +21,17 @@ venues:
...
+      # conference id
+      venue: MLSys
+      # conference full name
+      name: The 4rd Conference on Machine Learning and Systems
+      # date for sorting
+      date: 2021-03-02
```
Comment (Aetf) [inline source/publications/index.md:30]:
```suggestion
      date: 2021-04-05
```

File: source/_data/SymbioticLab.bib [hunk 5]
```diff
@@ -626,3 +629,25 @@ @InProceedings{kayak:nsdi21
     How cloud applications should interact with their data re-mains an active area of research. Over the last decade, manyhave suggested relying on a key-value (KV) interface to inter-act with data stored in remote storage servers, while othershave vouched for the benefits of using remote procedure call(RPC). Instead of choosing one over the other, in this paper,we observe that an ideal solution must adaptively combineboth of them in order to maximize throughput while meetingapplication latency requirements. To this end, we proposea new system called Kayak that proactively adjusts the rateof requests and the fraction of requests to be executed usingRPC or KV, all in a fully decentralized and self-regulated man-ner. We theoretically prove that Kayak can quickly convergeto the optimal parameters.  We implement a system proto-type of Kayak. Our evaluations show that Kayak achievessub-second convergence  and improves  overall throughputby 32.5%-63.4% for compute-intensive workloads and upto 12.2% for non-compute-intensive and transactional work-loads over the state-of-the-art.
   }
 }
+
+@InProceedings{fluid:mlsys21,
+  author    = {Peifeng Yu*, Jiachen Liu* and Mosharaf Chowdhury},
```
Comment (Aetf) [inline source/_data/SymbioticLab.bib:634]: Sadly there's no standard way to show equal contribution in bibtex... Adding `*` after the name will probably break standard styling/sorting/etc.

**Review by Aetf — CHANGES_REQUESTED** [review]

File: source/_data/SymbioticLab.bib (hunk 5 above)
Comment (AmberLJC) [reply source/_data/SymbioticLab.bib:634]: http://www.hansenlab.org/cv_bibliography_tex
Will
  author+an =	 {1=highlight; 2=highlight}
works?

Comment (Aetf) [reply source/_data/SymbioticLab.bib:634]: Probably. Let's do this. It's not rendered on the website, though. I need to implement that. But if someone copies the bibtex, it should work in their pdf.

**Pull request closed without merging**